apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeGlobalExplanationChart.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { toNumber } from "lodash";
+
+import { Chart, IChartElement } from "../../../../util/Chart";
+import { getComboBoxValue, selectComboBox } from "../../../../util/comboBox";
+import { ScatterChart } from "../../../../util/ScatterChart";
+import { IModelAssessmentData } from "../../IModelAssessmentData";
+
+const topKLabelReg = /^Top (\d+) features by their importance$/;
+function getTopKValue(): number {
+  const exec = topKLabelReg.exec(cy.$$("#TopKSliderContainer label").text());
+  if (!exec || !exec[1]) {
+    throw new Error("Cannot find top k label");
+  }
+  return toNumber(exec[1]);
+}
+
+export function describeGlobalExplanationChart<
+  TElement extends IChartElement
+>(props: { chart: Chart<TElement>; dataShape: IModelAssessmentData }): void {
+  describe("Global explanation chart", () => {
+    it("should have y axis label", () => {
+      cy.get('#FeatureImportanceBar div[class*="rotatedVerticalBox"]').should(
+        "contain.text",
+        "Aggregate feature importance"
+      );
+    });
+    it("should have x axis label", () => {
```

Comment:
so this all looks very similar to:

responsible-ai-widgets\apps\dashboard-e2e\src\describer\interpret\aggregateFeatureImportance\describeGlobalExplanationChart.ts

I am wondering if there is a better way to refactor this code, as it seems to be duplicating the other test code?  Or did I misunderstand?

Reply:
Couldn't you just use the other file directly here?

Reply:
oh, was this supposed to be a move and not a copy?  It looks like the old files were not removed.

Reply:
Yes, as for now it is copied and there are few changes in some files. And more will be added in future in similar directory structure. Ultimately files under dashboard-e2e/static dashboard tests will be removed once all real time notebook tests are in.